new_post.sh

```diff
@@ -0,0 +1,116 @@
+#!/bin/bash
+
+echo "This script will help you to initialize a new blog post."
+echo ""
+
+
+echo "1/ Category of the blog post"
+echo ""
+
+categories=("ai" "architecture" "cloud" "data" "dev" "general" "green" "mobile" "ops" "web")
+
+cat_id=0
+echo "  Blog post categories:"
+for cat in "${categories[@]}"
+do
+    echo "   [${cat_id}] ${cat}"
+    cat_id=$((cat_id+1))
+done
+
+echo ""
+echo -n "  Select a category for the blog post (0-9) [default: null]: "
+read cat_id_input
+if [[ -z "${cat_id_input}" ]]; then
+    echo "Error: Category is required"
+    exit 1
+fi
```

Comment:
is it possible to re-prompt if the value is not correct instead of exiting the sh ?

Reply:
so please delete this new_post.sh that will be introduced in the next PR;

Reply:
File removed